packages/charts/src/WrapLabel/wrapSVGText.js

```diff
@@ -10,10 +10,11 @@ export function computeMaxLabelDimension({ labelWidth, texts }) {
   document.body.appendChild(svg);
   svg.appendChild(textEl);
 
-  const text =
+  const text = String(
```

Comment:
Works but why not ES6 templates or even just plain [concatenation](https://stackoverflow.com/questions/29055518/are-es6-template-literals-faster-than-string-concatenation)?

Reply:
Will keep this in mind.